d2/src/main/java/com/linkedin/d2/xds/XdsClientImpl.java

```diff
@@ -440,27 +522,29 @@ public String toString()
     }
   }
 
-  private static final class DiscoveryResponseData
+  public static final class DiscoveryResponseData
   {
     private final ResourceType _resourceType;
     private final List<Resource> _resources;
+    private final List<String> _removedResources;
     private final String _nonce;
     @Nullable
     private final String _controlPlaneIdentifier;
 
-    DiscoveryResponseData(ResourceType resourceType, List<Resource> resources, String nonce,
-        @Nullable String controlPlaneIdentifier)
+    DiscoveryResponseData(ResourceType resourceType, List<Resource> resources, List<String> removedResources, String nonce,
+                        @Nullable String controlPlaneIdentifier)
```

Comment:
Watch the line length. Can you line this up to this:
```suggestion
    DiscoveryResponseData(
        ResourceType resourceType,
        List<Resource> resources,
        List<String> removedResources,
        String nonce,
        @Nullable String controlPlaneIdentifier
    )
```

Reply:
Done